spec/workers/pull_request_monitor/pr_branch_record_spec.rb

```diff
@@ -0,0 +1,106 @@
+require "spec_helper"
+
+RSpec.describe PullRequestMonitor::PrBranchRecord do
+  describe ".create" do
+    it "has git create a local pr branch" do
+      repo = spy("CommitMonitorRepo")
```

Comment:
Why spy vs double?  I never used spy before, and it seems inconsistent to introduce a new thing unless it has some different purpose.